packages/contracts/src/contracts/current/protocol/AssetProxy/MixinERC20Transfer.sol

```diff
@@ -0,0 +1,71 @@
+/*
+
+  Copyright 2018 ZeroEx Intl.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+*/
+
+pragma solidity ^0.4.24;
+pragma experimental ABIEncoderV2;
+
+import "../../utils/LibBytes/LibBytes.sol";
+import "../../tokens/ERC20Token/IERC20Token.sol";
+
+contract MixinERC20Transfer is
+    LibBytes
+{
+    // Id of this proxy.
+    uint8 constant PROXY_ID = 1;
```

Comment:
There have been some changes in the implementation of `transferFromInternal` for both proxies, so they no longer require `PROXY_ID`. We can move `PROXY_ID` back to the actual proxies after those changes have been merged into this branch.

Reply:
awww yissssss